cds/modules/deposit/static/templates/cds_deposit/types/project/form.html

```diff
@@ -71,7 +71,7 @@ <h3 class="mb-5 mt-0">
                 <span ng-if="!$ctrl.isAnyVideoAlreadyPublished()">
                   Select category and type, they will be applied to the project and all videos.
                 </span>
-                <span ng-if="$ctrl.isAnyVideoAlreadyPublished()">Category and type can be changed only if no video has been published yet.</span>
+                <span ng-if="$ctrl.isAnyVideoAlreadyPublished()">Category and type can be changed only if no video has been published nor report number reserved.</span>
```

Comment:
Just english kicked in. Probably we need to rename the phrase to `Category and type can be changed only if neither video has been published nor report number reserved.` :)

Reply:
Do we have an English speaker around?
Perhaps @costaflavio can help us with the language ☺️

Reply:
He is on holidays. @zzacharo can we merge anyway and eventually change it later if Flavio has a different opinion?

Reply:
@egabancho Zac agreed to merge, I was convincing! :)
I will merge!